### PR TITLE
chore: update docs on migrating project to Renku instructions

### PR DESCRIPTION
**Pull request by olevski**
The instructions to migrate an existing project from another repo to Renku have a typo in the git url.

Also the online documentation for GitLab says that you should be able to create a project by just doing `git push` just like the current documentation suggests. However this does not work very well. When you run the command first (without an existing repo) it first reports and error but it does end up creating an empty repo. Then you have to run `git push` again to get the code/data in there. Therefore I changed the instructions to not use this method and just ask the user to create the repo through the GitLab website.

I think the main problems with the errors from git push is that our GitLab version is a bit behind and this feature does not work very well in the version we have.

### CODE REVIEW

File: docs/tutorials/02_migrating_to_renku.rst [hunk 1]
```diff
@@ -94,19 +94,24 @@ Now your project is ready to be sent to the web platform!
 Push your project to renkulab
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Pushing your project to Renku's GitLab instance boils down to adding the right
-remote and pushing the code. Perform the following, replacing ``<namespace>`` with
-the username that appears on the renkulab-generated GitLab instance (unless you
-are pushing to a group), and ``<project-name>`` with the name you are giving your project.
+Before you can push your project to Renku you need to create a GitLab repository
```
Comment (ableuler) [inline docs/tutorials/02_migrating_to_renku.rst:97]: AFAIK you can create a project on GitLab by simply pushing, no need to create the the project through the UI first. However, the information about access management etc through gitlab is still very useful.

**Review by ableuler — CHANGES_REQUESTED** [review]
Thanks for making this part of the docs way more helpful!

File: docs/tutorials/02_migrating_to_renku.rst (hunk 1 above)
Comment (ableuler) [reply docs/tutorials/02_migrating_to_renku.rst:97]: Actually reading the PR description now - sorry about that.

**Review by ableuler — APPROVED** [review]

**Comment by olevski** [comment]
Just for completeness, the error that happens when you try to create a repo just by pushing is as follows:
```
(⎈ |switch-zh-dev:tasko) ➜  push-test-repo git:(master) git push origin master
remote:
remote: ========================================================================
remote:
remote: 500 Internal Server Error
remote:
remote: ========================================================================
remote:
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
 (⎈ |switch-zh-dev:tasko) ➜  push-test-repo git:(master) git push origin master
Enumerating objects: 3, done.
Counting objects: 100% (3/3), done.
Writing objects: 100% (3/3), 222 bytes | 222.00 KiB/s, done.
Total 3 (delta 0), reused 0 (delta 0)
remote:
remote:
remote: The private project tasko.olevski/push-test-repo was successfully created.
remote:
remote: To configure the remote, run:
remote:   git remote add origin git@renkulab.io:tasko.olevski/push-test-repo.git
remote:
remote: To view the project, visit:
remote:   https://renkulab.io/gitlab/tasko.olevski/push-test-repo
remote:
remote:
To renkulab.io:tasko.olevski/push-test-repo.git
 * [new branch]      master -> master
```

**Pull request merged**